src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy

```diff
@@ -44,7 +44,7 @@ class Liberty implements Plugin<Project> {
 
         project.task('installLiberty', type: InstallLibertyTask) {
             description 'Installs Liberty from a repository'
-            logging.level = LogLevel.INFO
+            logging.level = LogLevel.DEBUG
```

Comment:
Yes, this was intentional. This sets the log level. In order to check debug log which is set by 'logger.debug' in the code, the logging level needs to be lowered to DEBUG